layersvt/device_simulation.cpp

```diff
@@ -1019,6 +1179,35 @@ void JsonLoader::GetValue(const Json::Value &parent, const char *name, VkPhysica
     GET_VALUE(sparseProperties);
 }
 
+void JsonLoader::GetValue(const Json::Value &parent, const char *name, VkPhysicalDeviceMaintenance3PropertiesKHR *dest) {
```

Comment:
I'm guessing there's a good reason not to, but would it be possible to generate these from the XML? You might get a bunch of getters you don't care about, but then this would be one less thing to worry about updating.

Reply:
This is one of the points of pain when adding new extensions and query overrides to Devsim.

I plan on refactoring the code to make it more easily extendable after it's been updated to simulate Vulkan 1.2 devices, and I'll keep this suggestion in mind while doing that.